### PR TITLE
Remove Channels from the Application. Remove Wreqr as a dependency.

### PR DESCRIPTION
**Pull request by jmeas**
:wave: wreqr

:wave: app channel

### CODE REVIEW

**Comment by paulfalgout** [comment]
:tada: :balloon: :birthday:

File: api/application.yaml [hunk 1]
```diff
@@ -216,7 +177,7 @@ functions:
           
           MyApp.start(options);
           ```
-
```
Comment (samccone) [inline api/application.yaml]: watch the deleting of WS here .. because in yaml it is needed

File: bower.json [hunk 2]
```diff
@@ -37,7 +37,6 @@
     "backbone": "1.0.0 - 1.1.2",
     "underscore": "1.4.4 - 1.6.0",
     "backbone.babysitter": "^0.1.0",
-    "backbone.wreqr": "^1.0.0",
```
Comment (samccone) [inline bower.json]: :wave:

**Comment by samccone** [comment]
this commit should add in radio, because the tests really will not change so much

**Comment by jmeas** [comment]
> this commit should add in radio, because the tests really will not change so much

Let's hold off on adding Radio until we introduce a PR that actually requires that we use Radio. I think that if the library doesn't **need** it, then it should be an extension that we officially promote, like `ngRoute`. We can document it in our documentation and go the whole nine yards with telling people about it, but just not bundling it up by default.

**Comment by megawac** [comment]
^ +100

**Comment by samccone** [comment]
yeah I gotcha @jmeas  after a chat

**Comment by samccone** [comment]
:+1:

**Comment by thejameskyle** [comment]
:+1:

**Pull request merged**